docker/Dockerfile-debug

```diff
@@ -2,10 +2,10 @@
 
 FROM golang:latest AS golang
 ENV GOPROXY=https://goproxy.io,direct
-RUN CGO_ENABLED=0 go get -ldflags '-s -w -extldflags -static' github.com/go-delve/delve/cmd/dlv
+RUN export DELVETAG=$(curl -s https://api.github.com/repos/go-delve/delve/releases/latest | grep "tag_name" | cut -f2 -d':' | tr -d ' ",') && CGO_ENABLED=0 go install -ldflags '-s -w -extldflags -static' github.com/go-delve/delve/cmd/dlv@$DELVETAG
```

Comment:
Any reason why this couldn't have been `go install github.com/go-delve/delve/cmd/dlv@latest`?

Reply:
The reason is that I didn't know this is a thing... hahaha